index.js

```diff
@@ -598,7 +598,7 @@ class GithubScm extends Scm {
         const command = [];
 
         command.push(
-            "export SD_GIT_WRAPPER=\"$(if [[ `uname` = 'Darwin' || $SD_HAB_ENABLED ]]; " +
+            "export SD_GIT_WRAPPER=\"$(if [[ `uname` = 'Darwin' || $SD_HAB_ENABLED = 'False' ]]; " +
```

Comment:
can you change this to `if [[ `uname` = 'Darwin' || $SD_HAB_ENABLED != 'true` ]]` so that we can be consistent with check used in `launcher` https://github.com/screwdriver-cd/launcher/blob/master/Docker/launcher_entrypoint.sh#L56